### DISCUSSION TITLE
How to use fade-out animation with Vue Transition component?

### DISCUSSION THREAD

**Discussion by ZiuChen** [xsjcTony/unocss-preset-animations#6] [Q&A]
When the `showLabel` variable changes from `false` to `true`, the fade-in animation displays correctly. However, when `showLabel` changes to false, the element disappears immediately without applying the fade-out animation.

```vue
<span v-show="showLabel" class="animate-in fade-in">
  {{ scale.toFixed(1) }}x
</span>
```

If I want to use it with the Transition component, how should I specify the `name` prop for the Transition?

```vue
<Transition>
  <span v-show="showLabel" class="animate-in fade-in">
    {{ scale.toFixed(1) }}x
  </span>
</Transition>
```

**Comment by xsjcTony** [comment]
I haven't been using Vue's `<Transition />` component a lot, but based on my knowledge, there are 2 ways to do so:
- Specify a transition name, and use `@apply` for those classnames in `<style>` block
- Use https://vuejs.org/guide/built-ins/transition.html#custom-transition-classes to give it a try. the mechanism should be similar to Animate.css in the example, so I'd suggest that way.
